src/main/java/randoop/main/GenTests.java

```diff
@@ -537,6 +537,12 @@ public boolean handle(String[] args) {
 
     JavaFileWriter javaFileWriter = new JavaFileWriter(junit_output_dir);
 
+    Report report = new Report();
+    List<ExecutableSequence> allSequences = explorer.getRegressionSequences();
+    allSequences.addAll(explorer.getErrorTestSequences());
+    report.setTotalTests(allSequences.size());
```

Comment:
I was wondering if this parameter could be only a local variable on method generateReport(). I'd say it because it is only used in this particular method.